index.js

```diff
@@ -136,36 +191,49 @@ function nowServing(line) {
 }
 
 const line = []
+var ticketNum = 0
 
 // ***** Scope & Closures - Question 1 *****
+// *** Uncomment the lines below to test
+console.log("%cScope & Closures - Question 1", "color: red")
 
+function takeATicketNumber (line) {
 
-// *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+  ticketNum += 1
+  return `Welcome. You are ticket number ${ticketNum}.`
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+}
```

Comment:
good. Would use `let` on line 194.

also you are missing pushing the ticketNumber into the line

```
let ticketNumber = 0

function takeATicketNumber(line) {
  ticketNumber += 1
  line.push(ticketNumber)
  return `Welcome. You are ticket number ${ticketNumber}`
}
```

Reply:
this way the line variable is getting updated so we can keep track of all the tickets.